pkg/network/ephemeral_linux.go

```diff
@@ -34,12 +35,14 @@ func IsPortInEphemeralRange(p uint16) EphemeralPortType {
 
 	low, hi, err := ephemeralIntPair.Get()
 	if err == nil {
+		mu.Lock()
```

Comment:
Instead of adding the mutex, you can move setting `ephemeralLow` and `ephemeralHigh` to the `sync.Once.Do` call.